lib/prefix.h

```diff
@@ -296,15 +296,17 @@ union prefixconstptr {
 #endif /* INET_ADDRSTRLEN */
 
 #ifndef INET6_ADDRSTRLEN
+/* dead:beef:dead:beef:dead:beef:dead:beef + \0 */
 #define INET6_ADDRSTRLEN 46
 #endif /* INET6_ADDRSTRLEN */
 
 #ifndef INET6_BUFSIZ
-#define INET6_BUFSIZ 51
+#define INET6_BUFSIZ 53
 #endif /* INET6_BUFSIZ */
 
 /* Maximum prefix string length (IPv6) */
-#define PREFIX_STRLEN 51
+/* dead:beef:dead:beef:dead:beef:dead:beef:255.255.255.255 + / + 65535 + \0 */
```

Comment:
An IPv6 prefix cannot have a `/65535` prefix length.  Whatever application is using a >=256 prefix length is going to have some completely different prefix output format.  We likely need to adjust PREFIX_STRLEN for that specific use case then.

Reply:
so what's the application that required prefix len  > 255?

Reply:
Yep, should have updated the comment. I changed this to 53 based on looking at the wrong `prefix2str` delegate; relooking at all of them, it seems PREFIX_STRLEN is already incorrect for existing code and should be even bigger. See `prefixevpn_es2str`, which I count as producing up to 69 characters:
```
[%d]:[%s]:[%s]/%d
```
1 + 10 + 3 + 30 + 3 + 15 + 2 + 5 = 69
(10 from `strlen(INT_MAX)`)
(30 from `esi_to_str`)
(15 from `strlen(255.255.255.255)`)
(5 from `strlen(65535)`)
(all others from `[`, `]`, `:` and `/` in the format string)

That said this oversight makes me feel really uncomfortable and I'm thinking we ought to make it 80 or something significantly bigger than what I just counted.